### PR TITLE
Support Scala 2.11.11

### PR DESCRIPTION
**Pull request by ktoso**
It seems 2.11.11 just works, good :)

### CODE REVIEW

**Comment by ktoso** [comment]
(for akka to upgrade https://github.com/akka/akka/pull/22746#issuecomment-295350071 )

**Comment by 2m** [comment]
Should we exclude 2.11.9 and 2.11.10 from cross scala versions?

**Comment by adriaanm** [comment]
There's no need to publish for those versions, as no-one else should be publishing using 2.11.9/10

**Comment by ktoso** [comment]
Ok, so excluding 2.11.9 and 10.

File: project/Build.scala [hunk 1]
```diff
@@ -24,9 +24,11 @@ object B extends Build {
     scalaVersion := crossScalaVersions.value.last,
     crossScalaVersions := {
       val latest210 = 6
-      val latest211 = 8
+      val latest211 = 11
       val latest212 = 2
+      val skipVersions = Set("2.11.9", "2.11.10")
```
Comment (ktoso) [inline project/Build.scala]: That's what you meant @2m right?
Please merge if looks good

Comment (adriaanm) [inline project/Build.scala]: Thanks for going the extra mile to compensate for my dud releases.

**Pull request merged**